src/include/hip/rpp_hip_common.hpp

```diff
@@ -1429,6 +1466,81 @@ __device__ __forceinline__ void rpp_hip_layouttoggle24_pln3_to_pkd3(T *pixpln3Pt
     *pixpln3Ptr_T24 = pixpkd3_T24;
 }
 
+// ------------------------- Set 8 - Loads to uint / int --------------------------
+
+__device__ __forceinline__ void rpp_hip_load8_to_uint8(uchar *srcPtr, d_uint8 *srcPtr_ui8)
+{
+    d_uchar8 src_uc8;
+    *(d_uchar8_s *)&src_uc8 = *(d_uchar8_s *)srcPtr;
+
+    srcPtr_ui8->ui4[0] = rpp_hip_pack_uint4(src_uc8.uc4[0]);
+    srcPtr_ui8->ui4[1] = rpp_hip_pack_uint4(src_uc8.uc4[1]);
+}
+
+__device__ __forceinline__ void rpp_hip_load8_to_int8(schar *srcPtr, d_int8 *srcPtr_i8)
+{
+    d_schar8_s src_sc8;
+    *(d_schar8_s *)&src_sc8 = *(d_schar8_s *)srcPtr;
+
+    rpp_hip_pack_int8(&src_sc8, srcPtr_i8);
+}
+
+__device__ __forceinline__ void rpp_hip_load24_pln3_to_uint24_pln3(uchar *srcPtr, uint increment, d_uint24 *srcPtr_ui24)
+{
+    d_uchar24 src_uc24;
+    *(d_uchar8_s *)&src_uc24.uc8[0] = *(d_uchar8_s *)srcPtr;
+    srcPtr += increment;
+    *(d_uchar8_s *)&src_uc24.uc8[1] = *(d_uchar8_s *)srcPtr;
+    srcPtr += increment;
+    *(d_uchar8_s *)&src_uc24.uc8[2] = *(d_uchar8_s *)srcPtr;
+
+    srcPtr_ui24->ui4[0] = rpp_hip_pack_uint4(src_uc24.uc4[0]);    // write R00-R03
+    srcPtr_ui24->ui4[1] = rpp_hip_pack_uint4(src_uc24.uc4[1]);    // write R04-R07
+    srcPtr_ui24->ui4[2] = rpp_hip_pack_uint4(src_uc24.uc4[2]);    // write G00-G03
+    srcPtr_ui24->ui4[3] = rpp_hip_pack_uint4(src_uc24.uc4[3]);    // write G04-G07
+    srcPtr_ui24->ui4[4] = rpp_hip_pack_uint4(src_uc24.uc4[4]);    // write B00-B03
+    srcPtr_ui24->ui4[5] = rpp_hip_pack_uint4(src_uc24.uc4[5]);    // write B04-B07
+}
+
+__device__ __forceinline__ void rpp_hip_load24_pln3_to_int24_pln3(schar *srcPtr, uint increment, d_int24 *srcPtr_i24)
+{
+    d_schar24_s src_sc24;
```

Comment:
please remove this data_type and follow the previous comment

Reply:
same as above